vendor.conf

```diff
@@ -117,12 +117,12 @@ github.com/googleapis/gax-go                        317e0006254c44a0ac427cc52a0e
 google.golang.org/genproto                          694d95ba50e67b2e363f3483057db5d4910c18f9
 
 # containerd
-github.com/containerd/containerd                    7c1e88399ec0b0b077121d9d5ad97e647b11c870
-github.com/containerd/fifo                          a9fb20d87448d386e6d50b1f2e1fa70dcf0de43c
-github.com/containerd/continuity                    aaeac12a7ffcd198ae25440a9dff125c2e2703a7
-github.com/containerd/cgroups                       4994991857f9b0ae8dc439551e8bebdbb4bf66c1
+github.com/containerd/containerd                    36cf5b690dcc00ff0f34ff7799209050c3d0c59a # v1.3.0
+github.com/containerd/fifo                          bda0ff6ed73c67bfb5e62bc9c697f146b7fd7f13
+github.com/containerd/continuity                    f2a389ac0a02ce21c09edd7344677a601970f41c
```

Comment:
not a big deal, but this should be 75bee3e2ccb6402e3a986ab8bd3b17003fc0fdec (see buildkit's go.mod). The only difference is https://github.com/containerd/continuity/compare/f2a389ac0a02ce21c09edd7344677a601970f41c...master#diff-baa2046d08674df161187d3f68336ff6R83

Reply:
I think I aligned it to https://github.com/containerd/containerd/blob/v1.3.0/vendor.conf#L7, given that this was listed under "containerd" dependencies. I can open a PR in containerd to update it there.